Content.Client/Administration/UI/SpawnExplosion/ExplosionDebugOverlay.cs

```diff
@@ -35,6 +36,8 @@ public ExplosionDebugOverlay()
 
         var cache = IoCManager.Resolve<IResourceCache>();
         _font = new VectorFont(cache.GetResource<FontResource>("/Fonts/NotoSans/NotoSans-Regular.ttf"), 8);
+
+        _transform = _entityManager.System<SharedTransformSystem>();
```

Comment:
I've run into issues resolving entitysystems in overlay constructors myself, but since it passes tests this is probably fine.

Reply:
I've tested it and it works fine